packages/showcases/svelte/stories/components/indicators/VtmnRating/VtmnRating.stories.svelte

```diff
@@ -0,0 +1,198 @@
+<script>
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnRating } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/indicators/rating.csf';
+  const argTypes = {
+    name: {
+      type: { name: 'string', required: false },
+      description: 'Name used on the input radio',
+      defaultValue: 'rating',
+      control: {
+        type: 'text',
+      },
+    },
+    emphasis: {
+      type: { name: 'boolean', required: false },
+      description: 'The variant of the rating.',
+      defaultValue: false,
+      control: {
+        type: 'boolean',
+      },
+    },
+    size: {
+      type: { name: 'string', required: false },
+      description: 'Size of the rating.',
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium'],
+      },
+    },
+    disabled: {
+      type: { name: 'boolean', required: false },
+      description: 'Whether the rating is disabled.',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    readOnly: {
+      type: { name: 'boolean', required: false },
+      description: 'Whether the rating is read only.',
+      defaultValue: true,
+      control: { type: 'boolean' },
+    },
+    compact: {
+      type: { name: 'number', required: false },
+      description: 'How many stars to display.',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    rating: {
+      type: { name: 'boolean', required: false },
+      description: 'How many stars are filled.',
+      defaultValue: 0,
+      control: { type: 'range', min: 0, max: 5, step: 0.25 },
```

Comment:
I think a step of 0.5 would be ok because we don't show quarter star

Reply:
Steps of 0.25 would have allowed to see that between 0.25 and 0.75, we have a 1/2 star